app/routes.js

```diff
@@ -0,0 +1,97 @@
+Project = require('./models/project');
+Employee = require('./models/employee')
+module.exports = function(app) {
+	app.get('/', function(req, res) {
+		res.send('Hello');
+	});
+	app.get('/api/project', function(req, res) {
+		Project.getprojects(function(err, project) {
+			if(err) throw err;
+			res.json(project);
+		});
+	});
+	app.post('/api/project', function(req, res) {
+		var project = req.body 
+		Project.addproject(project, function(err, project) {
+			if(err) throw err;
+			res.json(project);
+		});
+	});
+	app.get('/api/project/:_id', function(req, res) {
+		var id = req.params._id
+		Project.getprojectbyid(id, function(err, project) {
+			if(err) throw err;
+			res.send(project).json();
+		});
+	});
+	app.put('/api/project/:_id', function(req, res) {
+		var id = req.params._id;
+		Project.findById(id, function(err, project) {
+			if(err) throw err;
+			var body = req.body;
```

Comment:
Why do you need to create an extra variable `body` when you're just using it at one place? Just directly write `req.body`.